### PR TITLE
Added transfers endpoint

### PR DESCRIPTION
**Pull request by jamtur01**
### Description

Added the transfers endpoint.

### Checklist

- [X] Code compiles correctly
- [X] Created tests which fail without the change (if applicable)
- [X] All lint and unit tests passing
- [X] Extended the README / documentation, if necessary

### CODE REVIEW

**Review by bgrnwd — APPROVED** [review]

**Comment by bgrnwd** [comment]
@jamtur01 Thanks for the PR!

**Comment by jamtur01** [comment]
@bgrnwd Done - I also added another endpoint to return all leagues. Happy to pull it out too.

File: docs/api.md [hunk 1]
```diff
@@ -77,6 +68,27 @@ Dates are formatted like so: `20200814`
 
 <https://www.fotmob.com/api/matchDetails?matchId={matchId}>
 
+### World News
+
+`GET`
+
+<https://www.fotmob.com/api/transfer>
```
Comment (bgrnwd) [inline docs/api.md:75]:
```suggestion
<https://www.fotmob.com/api/worldnews>
```

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "fotmob",
-  "version": "2.2.1",
+  "version": "2.2.2",
```
Comment (bgrnwd) [inline package.json:3]: My bad, I should have been more explicit. I think this should be `2.3.0`

**Review by bgrnwd — CHANGES_REQUESTED** [review]

**Comment by bgrnwd** [comment]
> @bgrnwd Done - I also added another endpoint to return all leagues. Happy to pull it out too.

Nope, happy to have it. Just reviewed the PR and suggested a couple minor changes.

**Comment by ItamarShDev** [comment]
Just pointing out that is seems the fomatter settings of the OP is changing all files.
We need to make sure the formating is consistent

**Comment by bgrnwd** [comment]
> Just pointing out that is seems the fomatter settings of the OP is changing all files. We need to make sure the formating is consistent

I did notice that. I was hoping the `.prettierrc` and `.eslintrc.json` would help with that but I did see your additional linting changes in your most recent PR. I think those will be good to add.

**Review by bgrnwd — APPROVED** [review]

**Pull request merged**